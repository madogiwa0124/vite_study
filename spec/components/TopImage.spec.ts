import { mount } from "@vue/test-utils";
import Component from "@js/components/topImage.vue";

describe("components/topImage.vue", () => {
  describe("初期表示", () => {
    it("snapshot", () => {
      const props = { alt: "test alt" };
      const wrapper = mount(Component, { propsData: props });
      expect(wrapper.element).toMatchSnapshot();
    });
  });
});
