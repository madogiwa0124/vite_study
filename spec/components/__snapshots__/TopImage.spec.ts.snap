// Vitest Snapshot v1

exports[`components/topImage.vue > 初期表示 > snapshot 1`] = `
<div
  class="top-image"
>
  <img
    alt="test alt"
    height="350"
    src="/src/images/noimage.png"
    width="350"
  />
</div>
`;
